test/complex_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule ComplexTest do
+  use ExUnit.Case
+  # Calculated by trial and error :(
+  @epsilon 1.0e-15
+
+  test "can be created with numbers representing real and imaginary parts" do
+    z = Complex.new(1, 2.0)
+    assert z.re == 1
+    assert z.im == 2
+  end
+
+  test "can be created from a 2-tuple of real and imaginary parts" do
+    assert Complex.new(1, 2) == Complex.new({1, 2})
```

Comment:
This was intended to be syntactic sugar to make passing "bare values" into pipelines easier.

I stopped short of converting all the functions to do this, because at that point it would have looked fairly pointless to have a %Complex{} structure type.

What do you think?